projects/ssr-tests/src/environments/custom-test-environment.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Event, State } from 'jest-circus';
+import { TestEnvironment as NodeEnvironment } from 'jest-environment-node';
+import { getRawLogsPretty } from '../utils/log.utils';
+
+/**
+ * This is a custom Jest environment that adds the SSR logs to the test errors.
+ * The logs are added to the error `cause` property for failed tests,
+ * so those logs will be printed by Jest along with the test error.
+ */
+export default class CustomTestEnvironment extends NodeEnvironment {
```

Comment:
`query:` How Jest knows that `CustomTestEnvironment` shoudld be used? I didn't find any usage of this class. Is it file-based config, so Jest knows that it should use default export from `environments/custom-test-environment`?